app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @publisher = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(params[:comment].permit(:Text))
+    @comment.Author_id = current_user.id
+    @comment.post_id = @post.id
+    if @comment.save
+      @comment.update_comments_counter
+      flash[:success] = 'Object successfully created'
+    else
+      flash[:error] = 'Something went wrong'
+    end
+    redirect_to user_post_path(@publisher, @post)
+  end
```

Comment:
# Optional

- [ ] This was also mandatory requirements. please make sure a comment is created on behalf of the currently logged in user (current_user) of the app.

- [ ] Remember also to add strong  parameters. You can refer to this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.)